src/lib/homestore_backend/replication_state_machine.hpp

```diff
@@ -166,6 +166,12 @@ class ReplicationStateMachine : public homestore::ReplDevListener {
     /// @brief Called when the replica set is being stopped
     void on_replica_stop() override;
 
+    /// @brief Called when the snapshot is being created.
+    /// @param snapshot - The snapshot object which contains the lsn and term;
+    void create_snapshot(homestore::repl_snapshot&) override {
```

Comment:
This is a place holder to allow HomeObject to be able to compile with HomeStore 6.3